packages/react-components/src/components/styles/ChatMessageComponent.styles.ts

```diff
@@ -0,0 +1,83 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import { mergeStyles, IStyle } from '@fluentui/react';
+import { CSSProperties } from 'react';
+
+export const chatMessageStyle = mergeStyles({
+  fontWeight: '600'
+});
+
+export const chatActionStyle = mergeStyles({
+  visibility: 'hidden',
+  width: '1rem',
+  height: '1rem',
+  cursor: 'pointer',
+  [`.${chatMessageStyle}:hover &`]: {
+    visibility: 'visible'
+  },
+  '& svg': {
+    height: '1rem',
+    width: '1rem'
+  },
+  '&:hover svg': {
+    stroke: 'currentColor'
+  }
+});
+
+export const chatActionsCSS: IStyle = {
+  'ul&': {
+    boxShadow: '0 .2rem .4rem .1rem rgba(0, 0, 0, .1)',
```

Comment:
`theme.effects.elevation4`

Reply:
Not sure for this part, this shadow style is something we need to show the boundary of the hover button - if we make it another style, the boundary will be blurred a little bit, which is what I wanna avoid

Maybe we need a border or make color different from background, will keep it in the PR for now, and talk with AlexP about this, maybe I can find some solutions in the following PR